pages/1.12/release-notes/1.12.3/index.md

```diff
@@ -0,0 +1,150 @@
+
+## Mesos
+- DCOS-46554 - This change forces Mesos master to have port resources in every offer unless the offer contains disks. This helps to reduce the number of offers with no ports which are not useful for most frameworks.
+
+## Metrics
+- DCOS-47991, DCOS_OSS-4760 - In the MWST 1.13, there are gaps in various Mesos Grafana dashboards. Further investigation revealed that metrics were occasionally missing the `framework_name=<framework_name>` tag but with an additional `role_name=<framework_name>` tag. This exercise revealed a bug in the Mesos input plugin in Telegraf that may be causing this issue. The solution is to fix the hashing function to avoid potentially mapping fields to wrong tags.
+
+- DCOS_OSS-4624 - Currently, you cannot view the disk usage of a Mesos persistent volume (only Mesos sandbox is shown) for instance and the disk space used by Prometheus. This release adds the missing container metrics such as `DiskStatistics`, `Perf`, `NetTrafficControlStatistics` and `NetSNMPStatistics` from `mesos-go` and also adds all available `blkio stats`.
+
+## Networking
+- COPS-3279, COPS-3576, DCOS-37703, DCOS-37703, DCOS-39703 - Service endpoint values and service address-based statistics return the correct number of successful and failure connections when you enable the `statsd` metrics input plugin and view backend activity.
+
+[enterprise]
+## Security
+[/enterprise]
+- DCOS-46381, DCOS-47348 - The Marathon app definition format has changed from [1.4 to 1.5](https://github.com/mesosphere/marathon/blob/master/docs/docs/upgrade/network-api-migration.md#example-definitions). Current Admin Router code supports only v1.4 app definition. The Admin Router is not able to expose apps requiring `ip-per-container` feature at `/service/` endpoint. This release adds a routing logic for Marathon v1.5 app definition. 
+
+- DCOS-47687 - The ZooKeeper snapshot and log files contain sensitive data and are readable by any user on a master node, so it is important to control permissions for ZooKeeper data directories. This fix ensures that `/var/lib/dcos/exhibitor/zookeeper` is owned by `dcos_exhibitor` and only has owner permissions.
+
+# Known Issues and Limitations
+This section covers any known issues or limitations that don’t necessarily affect all customers, but might require changes to your environment to address specific scenarios. The issues are grouped by feature, functional area, or component. Where applicable, issue descriptions include one or more issue tracking identifiers.
+
+## Mesos
+- COPS-4104 - This release fixes an issue that caused container and agent recovery to fail under the following circumstances:
```

Comment:
I am confused why this item is listed under 'Known issues and limitations',  from the fix version I believe this issue is fixed in 1.12.3.